### DISCUSSION TITLE
No new tags or version release for long time

### DISCUSSION THREAD

**Discussion by ntimilsina-fieldwire** [jnunemaker/httparty#731] [General]
Why no new tags since  v0.18.1 - June 10, 2020 or version release since v0.17.3 - Dec 16, 2019

**Comment by jnunemaker** [comment]
Whoops! Pretty stable at this point. Sometimes I merge and then forget to release. I'll get a new one out soon™. 😄

**Reply by ntimilsina-fieldwire** [reply]
Looking forward to that @jnunemaker

**Comment by jnunemaker** [comment]
0.19.0 is out.

**Comment by lacostej** [comment]
The latest release as displayed on the home page is 0.17.3. https://github.com/jnunemaker/httparty/releases

Something must be updated on github side.
